prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -63,6 +67,11 @@ class ReportFunction(
 ) : RequestFunction(workflowEngine),
     Logging {
 
+        enum class IngestionMethod {
```

Comment:
This is something that can get moved later to submission microservice along with SFTP function and then we can remove the others, so I'm okay with it being defined here instead of shared repo.